Entities/Items/Explosives/KegVoodoo.as

```diff
@@ -69,6 +69,8 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream @params)
 		this.SetLight(true);
 		this.SetLightRadius(this.get_f32("explosive_radius") * 0.5f);
 		this.getSprite().SetEmitSound("/Sparkle.ogg");
+		this.getSprite().SetEmitSoundSpeed(1.0f);
+		this.getSprite().SetEmitSoundVolume(1.0f);
```

Comment:
Why is this required?

Reply:
'Fixed weird fuse sound when relighting keg after it was frozen by admin'
Probably to fix this ^